pyfar/plot/two_d.py

```diff
@@ -5,6 +5,98 @@
 from . import _interaction as ia
 
 
+def time2d(signal, dB=False, log_prefix=20, log_reference=1, unit=None,
+           points=None, orientation="vertical",
```

Comment:
What's the idea behind `orientation="vertical"` as default?

Reply:
That is simply my preference, because I had to chose some default. Would you prefer `'horitzontal'` as the default?

Reply:
I do not have a preference, it's fine.